src/core/Lock/Async.pm6

```diff
@@ -79,9 +80,14 @@ my class Lock::Async {
                 }
             }
         }
+#?endif
+#?if js
+                    return KEPT-PROMISE;
```

Comment:
This may be nit-picking, but I'd just indent this one level, not the same as the other `return`s.

Reply:
Done